compatibility-verifier/sample-test-suite/pre-controller-upgrade.yaml

```diff
@@ -20,6 +20,12 @@
 # Operations to be done.
 description: Operations to be run before Controller upgrade
 operations:
+  - type: tableOp
```

Comment:
you need to create the table after creating the kafka topic

Reply:
updated!